package.json

```diff
@@ -44,6 +44,7 @@
     "glob": "^7.1.7",
     "husky": "^6.0.0",
     "is-ci": "^3.0.0",
+    "is-regexp": "^3.0.0",
```

Comment:
I added this dep because it's a simple one-line function that uses ES modules (so we don't need `@rollup/plugin-commonjs`), and I wanted to test a third-party package import.

I would have used something more fun like `cowsay` but it wasn't simple and didn't support ES modules well.

Reply:
Instead of adding a dependency for testing purposes only, I would rather prefer handcrafting a `node_modules` directory under `packages/@lwc/rollup-plugin/src/__tests__/fixtures/simple_app_third_party_import/`. It's something we are already doing for `packages/@lwc/rollup-plugin/src/__tests__/fixtures/multi_version`. 

Note that `node_modules` directories under `packages/@lwc/module-resolver/src/__tests__/` are actually check into git ([.gitignore](https://github.com/salesforce/lwc/blob/master/.gitignore#L21)).

Reply:
OK, done. Although I had to modify the `.gitignore`, since there it looks like the `multi_version` `node_modules` were actually force-committed to Git.

Reply:
True good catch it's not the same project. My bad.